### PR TITLE
Upstream changes 20160418

### PR DESCRIPTION
**Pull request by johnsyweb**
## Context

I'd like to use a later version of Nokogiri to test a fix for https://github.com/sparklemotion/nokogiri/issues/1426
## Change
- Pull in upstream changes
- Relax nokogiri version
## Considerations

Should we send these changes upstream?
## Cc

@mariovisic & @jacobbednarz

### CODE REVIEW

**Comment by jacobbednarz** [comment]
Makes sense! :ship:

File: lib/wraith/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Wraith
-  VERSION = "3.1.2"
+  VERSION = "3.1.4"
```
Comment (asellitt) [inline lib/wraith/version.rb:2]: Mmm, pi!

**Comment by asellitt** [comment]
+1

File: lib/wraith/version.rb (hunk 1 above)
Comment (johnsyweb) [inline lib/wraith/version.rb:2]: π!

**Comment by mariovisic** [comment]
👍

**Comment by mariovisic** [comment]
Can I also suggest sending a PR to the project with the relaxed nokogiri version and an explanation as to why it's useful?

**Comment by johnsyweb** [comment]
@mariovisic : You can indeed! https://github.com/BBC-News/wraith/pull/425

**Comment by johnsyweb** [comment]
@mariovisic BBC-News#425 has been merged. Are we safe to move back off our fork?

**Pull request merged**